vue/sbc-common-components/src/components/NotificationPanel.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    v-if="showNotifications"
+    v-on:clickout="emitClose()"
+  >
+    <v-overlay></v-overlay>
+    <v-navigation-drawer
+      right
+      app
+      :width="440"
+      style="top: 70px; max-height: calc(100% - 0);"
+    >
+      <v-app-bar
+        flat
+        outlined
+        color="#FCBA19"
+      >
+        <v-toolbar-title class="toolbar-title">What's New at BC Registries</v-toolbar-title>
+        <v-spacer></v-spacer>
+        <v-btn
+          icon
+          large
+          class="dialog-close"
+          @click="emitClose()"
+        >
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-app-bar>
+      <v-list flat>
+        <v-list-item-group color="primary">
+          <template v-for="(item, i) in notifications">
+            <v-list-item :key="i">
+              <v-row dense>
+                <v-col class="d-flex" cols="1">
+                  <span :class="!item.read && (item.priority ? 'dot-red' : 'dot-blue')">
+                  </span>
+                </v-col>
+                <v-col>
+                <v-list-item-content>
+                  <v-list-item-title class="font-weight-bold list-subtitle">{{item.title}}</v-list-item-title>
+                  <v-list-item-subtitle>{{item.date}}</v-list-item-subtitle>
+                  <v-spacer></v-spacer>
+                  <v-list-item-content v-html="item.description"></v-list-item-content>
+                </v-list-item-content>
+                </v-col>
+              </v-row>
+            </v-list-item>
+            <v-divider v-if="i < notifications.length - 1" :key="`${i}-divider`"></v-divider>
```

Comment:
Another time, you can simply render this divider here, and then use CSS (last-of-type or similar) to hide the last divider 😀

Reply:
example?

Reply:
https://github.com/bcgov/namerequest/blob/9a6ae051254e97a29e75a38df947b140df3b0627/src/components/dialogs/receipts.vue#L99

Reply:
^^ That is if all the elements of the same type are together. You may need to use last-child or something else. Google it -- there are lots of examples.

Reply:
I will ignore this recommendation and revisit it later when I have time.

Reply:
As agreed, yes.